Build failure: rocdbgapi

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```console
[user@system]$ nix build github:NixOS/nixpkgs?ref=3016b4b15d13f3089db8a41ef937b13a9e33a8df#rocmPackages.rocdbgapi
error: builder for '/nix/store/isxfywxw3grx3vf4a6qvwh3yr0w97b02-rocdbgapi-6.3.3.drv' failed with exit code 2;
       last 25 log lines:
       > [9]
       > [10]) (./group__basic__group.tex
       > [11]
       > [12]
       > [13]
       > ! Undefined control sequence.
       > <argument> ...rtpbox {\@nextchar }\insert@pcolumn
       >                                                   \@endpbox \do@row@strut \o...
       > l.146 \begin{DoxyEnumFields}[2]{Enumerator}
       >
       > ?
       > ! Emergency stop.
       > <argument> ...rtpbox {\@nextchar }\insert@pcolumn
       >                                                   \@endpbox \do@row@strut \o...
       > l.146 \begin{DoxyEnumFields}[2]{Enumerator}
       >
       > !  ==> Fatal error occurred, no output PDF file produced!
       > Transcript written on refman.log.
       > Please consult refman.log to see the error messages
       > make[4]: *** [Makefile:12: refman.pdf] Error 1
       > make[3]: *** [CMakeFiles/doc.dir/build.make:77: doc/html/index.html] Error 2
       > make[3]: *** Deleting file 'doc/html/index.html'
       > make[2]: *** [CMakeFiles/Makefile2:120: CMakeFiles/doc.dir/all] Error 2
       > make[1]: *** [CMakeFiles/Makefile2:127: CMakeFiles/doc.dir/rule] Error 2
       > make: *** [Makefile:202: doc] Error 2
       For full logs, run:
         nix log /nix/store/isxfywxw3grx3vf4a6qvwh3yr0w97b02-rocdbgapi-6.3.3.drv
```



### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/301343437

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/6yq8kx24b077ngkwgaqrxdxh54w3lmh6-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/yhxwcanrwh0agzw79c934v3qr131qpbp-rocdbgapi-6.3.3-doc/share/doc/amd-dbgapi -DCMAKE_INSTALL_INFODIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/include -DCMAKE_INSTALL_SBINDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/2d2whnq221ja9gibrsxw3h88ksacj79i-rocmcxx/bin/strip -DCMAKE_RANLIB=/nix/store/2d2whnq221ja9gibrsxw3h88ksacj79i-rocmcxx/bin/ranlib -DCMAKE_AR=/nix/store/2d2whnq221ja9gibrsxw3h88ksacj79i-rocmcxx/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/g698q4ja77q3jihqdrqpa2ghmp3yzs0p-rocdbgapi-6.3.3 -DPCI_IDS_PATH=/nix/store/fahfp2sqndyp62n1h4ff9bb56fi22bs8-hwdata-0.396/share/hwdata -DCMAKE_INSTALL_BINDIR=bin -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_INSTALL_INCLUDEDIR=include
[0mCMake Deprecation Warning at CMakeLists.txt:21 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

[0m
-- The C compiler identification is Clang 18.0.0
-- The CXX compiler identification is Clang 18.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/2d2whnq221ja9gibrsxw3h88ksacj79i-rocmcxx/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/2d2whnq221ja9gibrsxw3h88ksacj79i-rocmcxx/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for backtrace.h
-- Looking for backtrace.h - not found
-- Using pci.ids file: /nix/store/fahfp2sqndyp62n1h4ff9bb56fi22bs8-hwdata-0.396/share/hwdata
-- Code Object Manager found at /nix/store/yb4z5iidvplziw01kfibgn51c31fcwv2-rocm-comgr-6.3.1/lib/cmake/amd_comgr.
-- Found Git: /nix/store/rzqvhv48m3nh8g3j4k6jmz6yqy8apr95-git-2.49.0/bin/git (found version "2.49.0")
[0mUsing CPACK_DEBIAN_PACKAGE_RELEASE local[0m
[0mUsing CPACK_RPM_PACKAGE_RELEASE local[0m
-- Found Doxygen: /nix/store/wmcm9ds6xiaiw8cry2xypdvr5lvqp2b0-doxygen-1.13.2/bin/doxygen (found version "1.13.2") found components: doxygen dot
-- Configuring done (0.5s)
-- Generating done (0.0s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025

[0m
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j32 SHELL=/nix/store/00zrahbb32nzawrmv9sjxn36h7qk9vrs-bash-5.2p37/bin/bash
[  9%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/agent.cpp.o[0m
[  9%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/architecture.cpp.o[0m
[ 13%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/callbacks.cpp.o[0m
[ 18%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/code_object.cpp.o[0m
[ 22%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/initialization.cpp.o[0m
[ 27%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/exception.cpp.o[0m
[ 36%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/event.cpp.o[0m
[ 36%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/dispatch.cpp.o[0m
[ 40%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/debug.cpp.o[0m
[ 45%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/displaced_stepping.cpp.o[0m
[ 50%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/logging.cpp.o[0m
[ 54%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/memory.cpp.o[0m
[ 59%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/process.cpp.o[0m
[ 63%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/os_driver.cpp.o[0m
[ 68%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/queue.cpp.o[0m
[ 72%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/register.cpp.o[0m
[ 77%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/utils.cpp.o[0m
[ 81%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/versioning.cpp.o[0m
[ 86%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/watchpoint.cpp.o[0m
[ 90%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/wave.cpp.o[0m
[ 95%] [32mBuilding CXX object CMakeFiles/amd-dbgapi.dir/src/workgroup.cpp.o[0m
[100%] [32m[1mLinking CXX shared library lib/librocm-dbgapi.so[0m
[100%] Built target amd-dbgapi
[100%] [34m[1mGenerating documentation[0m
warning: Tag 'TCL_SUBST' at line 257 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'CLANG_ASSISTED_PARSING' at line 1040 of file '/build/source/build/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_OPTIONS' at line 1048 of file '/build/source/build/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'COLS_IN_ALPHA_INDEX' at line 1066 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'HTML_TIMESTAMP' at line 1202 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'FORMULA_TRANSPARENT' at line 1483 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'LATEX_SOURCE_CODE' at line 1771 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'LATEX_TIMESTAMP' at line 1787 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'RTF_SOURCE_CODE' at line 1853 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOCBOOK_PROGRAMLISTING' at line 1951 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'PERL_PATH' at line 2131 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'CLASS_DIAGRAMS' at line 2144 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'MSCGEN_PATH' at line 2153 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_FONTNAME' at line 2195 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_FONTSIZE' at line 2202 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_TRANSPARENT' at line 2423 of file '/build/source/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
Doxygen version used: 1.13.2
Notice: Output directory '/build/source/build/doc/' does not exist. I have created it for you.
Searching for include files...
Searching for files in directory /build/source/include/
Searching for example files...
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for plantuml files...
Searching for files to exclude
Searching INPUT for files to process...
Reading and parsing tag files
Parsing files
Preprocessing /build/source/build/include/amd-dbgapi/amd-dbgapi.h...
Parsing file /build/source/build/include/amd-dbgapi/amd-dbgapi.h...
Building macro definition list...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Building concept list...
Computing nesting relations for classes...
Associating documentation with classes...
Associating documentation with concepts...
Associating documentation with modules...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Warn for undocumented namespaces...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Computing module relations...
Sorting lists...
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Setting anonymous enum type...
Computing dependencies between directories...
Generating citations page...
Counting members...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
Correcting members for VHDL...
Computing tooltip texts...
Generating style sheet...
Generating search indices...
Generating example documentation...
Generating file sources...
Generating code for file include/amd-dbgapi/amd-dbgapi.h...
Generating file documentation...
Generating docs for file include/amd-dbgapi/amd-dbgapi.h...
Generating page documentation...
Generating group documentation...
Generating dependency graph for group process_group
Generating dependency graph for group coredump_support
Generating class documentation...
Generating docs for compound amd_dbgapi_address_class_id_t...
Generating docs for compound amd_dbgapi_address_space_id_t...
Generating docs for compound amd_dbgapi_agent_id_t...
Generating docs for compound amd_dbgapi_architecture_id_t...
Generating docs for compound amd_dbgapi_breakpoint_id_t...
Generating docs for compound amd_dbgapi_callbacks_s...
Generating docs for compound amd_dbgapi_code_object_id_t...
Generating docs for compound amd_dbgapi_core_state_data_t...
Generating docs for compound amd_dbgapi_direct_call_register_pair_information_t...
Generating docs for compound amd_dbgapi_dispatch_id_t...
Generating docs for compound amd_dbgapi_displaced_stepping_id_t...
Generating docs for compound amd_dbgapi_event_id_t...
Generating docs for compound amd_dbgapi_process_id_t...
Generating docs for compound amd_dbgapi_queue_id_t...
Generating docs for compound amd_dbgapi_register_class_id_t...
Generating docs for compound amd_dbgapi_register_id_t...
Generating docs for compound amd_dbgapi_watchpoint_id_t...
Generating docs for compound amd_dbgapi_watchpoint_list_t...
Generating docs for compound amd_dbgapi_wave_id_t...
Generating docs for compound amd_dbgapi_workgroup_id_t...
Generating concept documentation...
Generating module documentation...
Generating namespace documentation...
Generating graph info page...
Generating directory documentation...
Generating dependency graph for directory include/amd-dbgapi
Generating index page...
Generating main page...
Generating page index...
Generating topic index...
Generating module index...
Generating module member index...
Generating namespace index...
Generating namespace member index...
Generating concept index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating graphical class hierarchy...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Combining RTF output...
Running plantuml with JAVA...
Running dot...
Generating dot graphs using 33 parallel threads...
Fontconfig error: Fontconfig error: Fontconfig error: Cannot load default config file: No such file: (null)Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Fontconfig error: Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)
Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)
Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)

Fontconfig error: 











Cannot load default config file: No such file: (null)

Fontconfig error: No writable cache directories

Fontconfig error: No writable cache directories

Fontconfig error: No writable cache directories

Fontconfig error: No writable cache directories
	/var/cache/fontconfig
	/var/cache/fontconfig
	/var/cache/fontconfig

Fontconfig error: No writable cache directories

Fontconfig error: No writable cache directories
	/var/cache/fontconfig

Fontconfig error: No writable cache directories
	/var/cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig

Fontconfig error: No writable cache directories

Fontconfig error: No writable cache directories
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/var/cache/fontconfig
	/var/cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/var/cache/fontconfig
	/var/cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
	/build/tmp.9MTpFcpPRj/.cache/fontconfig
Fontconfig error: Cannot load default config file: No such file: (null)
Fontconfig error: Fontconfig error: Cannot load default config file: No such file: (null)
Cannot load default config file: No such file: (null)
Fontconfig error: Cannot load default config file: No such file: (null)
Fontconfig error: Fontconfig error: Cannot load default config file: No such file: (null)Cannot load default config file: No such file: (null)

Running dot for graph 1/17
Running dot for graph 2/17
Running dot for graph 3/17
Running dot for graph 4/17
Running dot for graph 5/17
Running dot for graph 6/17
Running dot for graph 7/17
Running dot for graph 8/17
Running dot for graph 9/17
Running dot for graph 10/17
Running dot for graph 11/17
Running dot for graph 12/17
Running dot for graph 13/17
Running dot for graph 14/17
Running dot for graph 15/17
Running dot for graph 16/17
Running dot for graph 17/17
Patching output file 1/12
Patching output file 2/12
Patching output file 3/12
Patching output file 4/12
Patching output file 5/12
Patching output file 6/12
Patching output file 7/12
Patching output file 8/12
Patching output file 9/12
Patching output file 10/12
Patching output file 11/12
Patching output file 12/12
type lookup cache used 225/65536 hits=1480 misses=227
symbol lookup cache used 852/65536 hits=2312 misses=852
finished...
This is pdfTeX, Version 3.141592653-2.6-1.40.27 (TeX Live 2025/nixos.org) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./refman.tex
LaTeX2e <2025-06-01>
L3 programming layer <2025-05-26>

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/iftex/iftex.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/book.cls
Document Class: book 2025/01/22 v1.4n Standard LaTeX document class

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/bk10.clo))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/ifthen.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/array.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/array-2016-10-06.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/infwarerr/infwarerr.sty) (./doxygen.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/alltt.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/calc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/float/float.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/verbatim.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/xcolor/xcolor.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics-cfg/color.cfg)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics-def/pdftex.def)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics/mathcolor.ltx))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/colortbl/colortbl.sty) (./longtable_doxygen.sty) (./tabu_doxygen.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/varwidth/varwidth.sty)
==> First Aid for varwidth.sty applied!
)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/fancyvrb/fancyvrb.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics/keyval.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/tabularx.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tools/multicol.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/multirow/multirow.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hanging/hanging.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/iftex/ifpdf.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/adjustbox/adjustbox.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/xkeyval/xkeyval.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/xkeyval/xkeyval.tex
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/xkeyval/xkvutils.tex)))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/adjustbox/adjcalc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/adjustbox/trimclip.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics/graphicx.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics/graphics.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics/trig.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/graphics-cfg/graphics.cfg)))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/collectbox/collectbox.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/adjustbox/tc-pdftex.def)))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/amsfonts/amssymb.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/amsfonts/amsfonts.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/stackengine/stackengine.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/etoolbox/etoolbox.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/listofitems/listofitems.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/listofitems/listofitems.tex)))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/enumitem/enumitem.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/alphalph/alphalph.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/intcalc/intcalc.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/ulem/ulem.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/inputenc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/makeidx.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/textcomp.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/wasysym/wasysym.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/iftex/ifxetex.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/fontenc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/psnfss/helvet.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/psnfss/courier.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/geometry/geometry.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/iftex/ifvtex.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/changepage/changepage.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/fancyhdr/fancyhdr.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/natbib/natbib.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/tocloft/tocloft.sty)
Writing index file refman.idx

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/newunicodechar/newunicodechar.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/hyperref.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/kvsetkeys/kvsetkeys.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/kvdefinekeys/kvdefinekeys.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/pdfescape/pdfescape.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/ltxcmds/ltxcmds.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/pdftexcmds/pdftexcmds.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hycolor/hycolor.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/nameref.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/refcount/refcount.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/gettitlestring/gettitlestring.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/kvoptions/kvoptions.sty)))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/stringenc/stringenc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/pd1enc.def)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/puenc.def)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/backref.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/rerunfilecheck/rerunfilecheck.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/atveryend-ltx.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/uniquecounter/uniquecounter.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/bigintcalc/bigintcalc.sty))))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/url/url.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/gen
eric/bitset/bitset.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/base/atbegshi-ltx.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/hyperref/hpdftex.def)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/caption/caption.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/caption/caption3.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/caption/ltcaption.sty))
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/etoc/etoc.sty)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/psnfss/t1phv.fd)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/l3backend/l3backend-pdftex.def)
No file refman.aux.

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/con
text/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/epstopdf-pkg/epstopdf-base.sty
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/latexconfig/epstopdf-sys.cfg))
*geometry* driver: auto-detecting
*geometry* detected driver: pdftex
No file refman.toc.

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/amsfonts/umsa.fd)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/amsfonts/umsb.fd)
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/wasysym/uwasy.fd)
[1{/nix/store/784ps1b2q51w5q4dsaqwkcig1bmxa5r9-texlive-2025-env/share/texmf-var
/fonts/map/pdftex/updmap/pdftex.map}{/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzl
m-texlive-2025-env-texmfdist/fonts/enc/dvips/base/8r.enc}]
[2] (./index.tex
Chapter 1.

(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/psnfss/t1pcr.fd)
Overfull \hbox (34.10187pt too wide) in paragraph at lines 16--17
\T1/phv/m/n/10 placed step-ping mech-a-nism pro-vided by [] and []

Underfull \hbox (badness 1102) in paragraph at lines 16--17
\T1/phv/m/n/10 in con-junc-tion with the [] in sin-gle step mode. In or-der to 
de-ter-mine the lo-ca-tion

Underfull \hbox (badness 936) in paragraph at lines 16--17
\T1/phv/m/n/10 of stopped waves the client must read the ar-chi-tec-ture spe-ci
fic pro-gram counter reg-is-ter avail-able us-ing the

Underfull \hbox (badness 2213) in paragraph at lines 16--17
[] \T1/phv/m/n/10 query and ad-just it by the amount spec-i-fied by the

Underfull \hbox (badness 1205) in paragraph at lines 22--23
[]\T1/phv/m/n/10 A li-brary in-stance is de-fined as the pe-riod be-tween a cal
l to [] and a match-ing call to

[1]
(/nix/store/ac43fj1dj470jpb68njvrrrq7gx5xzlm-texlive-2025-env-texmfdist/tex/lat
ex/psnfss/ts1phv.fd)
[2]
[3]
[4]
Overfull \hbox (17.7223pt too wide) in paragraph at lines 106--107
[]\T1/phv/m/n/10 Generation of the [] event, the []

Underfull \hbox (badness 4013) in paragraph at lines 107--108
\T1/phv/m/n/10 front\TS1/phv/m/n/10 '\T1/phv/m/n/10 s [], [], or

Overfull \hbox (89.53241pt too wide) in paragraph at lines 107--108
[]\T1/phv/m/n/10 , or work-group\TS1/phv/m/n/10 '\T1/phv/m/n/10 s [],

Underfull \hbox (badness 10000) detected at line 111
[][][]

[5]
Underfull \hbox (badness 3492) in paragraph at lines 124--125
[]\T1/phv/m/n/10 Bus[]:Device.[]Function (BDF) Notation[]: []wiki.[]xen.[]org/w
iki/[]Bus[]:Device.[]Function[]_[]([]BDF)[]_[][]Notation[] ([][] \T1/pcr/m/n/10
 https[]

Overfull \hbox (82.38898pt too wide) in paragraph at lines 125--126
[]\T1/phv/m/n/10 HSA Plat-form Sys-tem Ar-chi-tec-ture Specification[]: [][][] 
\T1/pcr/m/n/10 https[]://hsafoundation.[]com/wp-content/uploads/2021/02/[]

Overfull \hbox (44.68877pt too wide) in paragraph at lines 126--127
[]\T1/phv/m/n/10 HSA Pro-gram-mer\TS1/phv/m/n/10 '\T1/phv/m/n/10 s Ref-er-ence 
Manual[]: [][][] \T1/pcr/m/n/10 https[]://hsafoundation.[]com/wp-content/upload
s/2021/02/[]

Underfull \hbox (badness 10000) in paragraph at lines 126--127
\T1/pcr/m/n/10 HSA-[]PRM-1.[]2.[]pdf[][][] \T1/phv/m/n/10 ([][] \T1/pcr/m/n/10 
https[]://hsafoundation.[]com/wp-content/uploads/2021/02/[]

Underfull \hbox (badness 2538) in paragraph at lines 129--130
[]\T1/phv/m/n/10 User Guide for AMDGPU LLVM Backend[]: []llvm.[]org/docs/[]AMDG
PUUsage.html[] ([][] \T1/pcr/m/n/10 https[]://llvm.[]

[6])
[7]
[8]
Chapter 2.
(./group__symbol__versions__group.tex
[9]
[10]) (./group__basic__group.tex
[11]
[12]
[13]
! Undefined control sequence.
<argument> ...rtpbox {\@nextchar }\insert@pcolumn 
                                                  \@endpbox \do@row@strut \o...
l.146 \begin{DoxyEnumFields}[2]{Enumerator}
                                           
? 
! Emergency stop.
<argument> ...rtpbox {\@nextchar }\insert@pcolumn 
                                                  \@endpbox \do@row@strut \o...
l.146 \begin{DoxyEnumFields}[2]{Enumerator}
                                           
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on refman.log.
Please consult refman.log to see the error messages
make[4]: *** [Makefile:12: refman.pdf] Error 1
make[3]: *** [CMakeFiles/doc.dir/build.make:77: doc/html/index.html] Error 2
make[3]: *** Deleting file 'doc/html/index.html'
make[2]: *** [CMakeFiles/Makefile2:120: CMakeFiles/doc.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:127: CMakeFiles/doc.dir/rule] Error 2
make: *** [Makefile:202: doc] Error 2
```

### Additional context

Blocks building `pytorch` or `llama.cpp` with ROCm support.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.15.3-cachyos, NixOS, 25.11 (Xantusia), 25.11pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - nixpkgs: `/nix/store/dax78flrvcdir20swzlcm64va09nh5x8-source`

### Notify maintainers

@NixOS/rocm-maintainers 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/rocm-modules/6/rocdbgapi/default.nix
+++ b/pkgs/development/rocm-modules/6/rocdbgapi/default.nix
@@ -37,6 +37,15 @@ let
         helvetic
         wasy
         courier
+        # FIXME: The following packages are used in the Doxygen table
+        # workaround, can be removed once
+        # https://github.com/doxygen/doxygen/issues/11634 is fixed, depending
+        # on what the fix is
+        tabularray
+        ninecolors
+        codehigh
+        catchfile
+        environ
       ]
     )
   );
@@ -58,6 +67,16 @@ stdenv.mkDerivation (finalAttrs: {
     rev = "rocm-${finalAttrs.version}";
     hash = "sha256-6itfBrWVspobU47aiJAOQoxT8chwrq9scRn0or3bXto=";
   };
+
+  # FIXME: remove once https://github.com/doxygen/doxygen/issues/11634 is resolved
+  # Applies workaround based on what was suggested in
+  # https://github.com/doxygen/doxygen/issues/11634#issuecomment-3027000655,
+  # but rewritten to use the `tabularray` LaTeX package. Unfortunately,
+  # verbatim code snippets in the documentation are not formatted very nicely
+  # with this workaround.
+  postPatch = ''
+    substituteInPlace doc/Doxyfile.in --replace 'LATEX_EXTRA_STYLESHEET =' 'LATEX_EXTRA_STYLESHEET = ${./override_doxygen_tables.sty}'
+  '';
 
   nativeBuildInputs = [
     cmake

--- a/pkgs/development/rocm-modules/6/rocdbgapi/override_doxygen_tables.sty
+++ b/pkgs/development/rocm-modules/6/rocdbgapi/override_doxygen_tables.sty
@@ -1,0 +1,89 @@
+\NeedsTeXFormat{LaTeX2e}
+
+% Packages used by this style file
+\RequirePackage{doxygen}
+\RequirePackage{tabularray}
+\UseTblrLibrary{varwidth}
+\RequirePackage{codehigh}
+\RequirePackage{environ}
+
+% Used by parameter lists
+\renewenvironment{DoxyParams}[2][]{%
+    \par%
+    \ifthenelse{\equal{#1}{}}%
+      {\begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,l]|X[-1,l]|}}}% name + description
+    {\ifthenelse{\equal{#1}{1}}%
+      {\begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,l]|X[-1,l]|X[-1,l]|}}}% in/out + name + desc
+      {\begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,l]|X[-1,l]|X[-1,l]|X[-1,l]|}}}% in/out + type + name + desc
+    }
+    \SetCell[c=2]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #2 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% Used for fields of simple structs
+\renewenvironment{DoxyFields}[1]{%
+    \par%
+    \begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,r]|X[-1,l]|X[-1,l]|}}%
+    \SetCell[c=3]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #1 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% Used for fields simple class style enums
+\renewenvironment{DoxyEnumFields}[2][]{%
+    \par%
+    \ifthenelse{\equal{#1}{2}}%
+      {\begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,r]|X[-1,l]|}}}%
+      {\begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,l]|X[-1,r]|X[-1,l]|}}}% with init value
+    \SetCell[c=2]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #2 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% Used by return value lists
+\renewenvironment{DoxyRetVals}[1]{%
+    \par%
+    \begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,r]|X[-1,l]|}}%
+    \SetCell[c=2]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #1 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% Used by exception lists
+\renewenvironment{DoxyExceptions}[1]{%
+    \par%
+    \begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|l[-1,r]|l[-1,l]|}}%
+    \SetCell[c=2]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #1 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% Used by template parameter lists
+\renewenvironment{DoxyTemplParams}[1]{%
+    \par%
+    \begin{longtblr}[l]{rowhead=1,measure=vbox,colspec={|X[-1,r]|X[-1,l]|}}%
+    \SetCell[c=2]{l} \hspace{-6pt}\bfseries\fontseries{bc}\selectfont\color{darkgray} #1 \\[1ex]%
+    \hline%
+}{%
+    \end{longtblr}%
+    \vspace{6pt}%
+}
+
+% NOTE: this is not a perfect workaround, and known to loose linebreaks, repeated spaces, etc
+\RenewEnviron{DoxyVerb}{%
+  \par%
+  \footnotesize%
+  \ExpandArgs{o}\fakeverb{\BODY}%
+  \normalsize%
+}

